homeassistant/components/smarttub/config_flow.py

```diff
@@ -7,14 +7,10 @@
 from homeassistant import config_entries
 from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
 
-from .const import DOMAIN
+# pylint complains because DOMAIN is only used in a keyword arg in the class declaration
+from .const import DOMAIN  # pylint: disable=unused-import
```

Comment:
```suggestion
from .const import DOMAIN
```

Reply:
Upgrade `pylint` if it still complains. Bug in pylint should be fixed in the latest